Utils/orwell.js

```diff
@@ -33,5 +34,20 @@ module.exports = {
     return new Promise((resolve, reject) => {
       r.db('DFB').table('analytics').get(user).run().then(resolve).catch(reject)
     })
+  },
+  roleUsers: (guild) => {
+    r.db("DFB").table("analytics").run().then((results) => {
+      results.forEach((row) => {
+        let totalDays = Object.keys(row.messages).length
+        let consecutiveDays = 7 // hardcoded until doug adds CD to schema
+        let member = guild.members.find(member => member.id === row.id)
+
+        Object.entries(roles).forEach(([key, role]) => {
+          if (totalDays && consecutiveDays >= role[0]) {
+            if (member) { member.assignRole(role[1]); console.log(`Gave ${member.id} ${key}`) }

```

Comment:
Shouldn't it rather send the user a message or log it in bot-log? Thought it doesn't do console outputs